protocol/message_validator.go

```diff
@@ -233,7 +233,13 @@ func ValidateReceivedChatMessage(message *protobuf.ChatMessage, whisperTimestamp
 		return errors.New("timestamp can't be 0")
 	}
 
-	if message.ContentType != protobuf.ChatMessage_DISCORD_MESSAGE {
+	if message.ContentType != protobuf.ChatMessage_DISCORD_MESSAGE && message.ContentType != protobuf.ChatMessage_IMAGE {
+		if err := ValidateText(message.Text); err != nil {
+			return err
+		}
+	}
+
+	if message.ContentType == protobuf.ChatMessage_IMAGE && message.Text != "" {
```

Comment:
It would be cleaner to add this condition to the previous like :
```go
if message.ContentType != protobuf.ChatMessage_DISCORD_MESSAGE || (message.ContentType == protobuf.ChatMessage_IMAGE && message.Text != "") {
```

Reply:
I'm thought about it so long... but I think you wrong :)

`||` that means `condition==true OR condition==true`

so if `message.ContentType == protobuf.ChatMessage_IMAGE && message.Text == ""` is make `message.ContentType != protobuf.ChatMessage_DISCORD_MESSAGE` as true :)